go.mod

```diff
@@ -72,7 +73,7 @@ require (
 	github.com/minio/minio-go/v6 v6.0.55
 	github.com/mitchellh/go-testing-interface v1.14.1 // indirect
 	github.com/mitchellh/mapstructure v1.2.3 // indirect
-	github.com/mkraft/gziphandler v1.1.2-0.20200509175700-73dc64f3ad90
+	github.com/mkraft/gziphandler v1.1.2-0.20200509175700-73dc64f3ad90 // indirect
```

Comment:
Why is this dependency still here if we adding in the `NYTimes` version? I have to say I don't have knowledge of how these specific dependencies are hooked into the application.

Reply:
Not sure why it was still showing up even though unused, but I have explicitly removed it now.